extensions/amp-date-picker/amp-date-picker.md

```diff
@@ -257,9 +257,9 @@ a name of `end-input` or `${id}-end-date` using the date picker's id. If either
 with an existing element in the form, an error is emitted.
 
 ```html
-<amp-date-picker 
-    type="range" 
-    start-input-selector="#a2" 
+<amp-date-picker
+    type="range"
+    start-input-selector="#a2"
```

Comment:
So, are these selectors okay with `#`?

Reply:
Oh, nvm.  Darn cold meds affect my parsing info.

Reply:
Ah yeah, there lots of selectors going on here. Feel better!